operators.py

```diff
@@ -3212,6 +3216,66 @@ class VIEW3D_OT_slvs_add_ratio(
     type = "RATIO"
 
 
+def set_all_constraints_visibility(context, visible=True, should_toggle=False):
+    constraint_lists = context.scene.sketcher.constraints.get_lists()
+    for constraint_list in constraint_lists:
+        for constraint in constraint_list:
+            if not hasattr(constraint, "visible"):
+                continue
+            constraint.visible = not constraint.visible if should_toggle else visible
+
+
+class View3D_OT_slvs_toggle_all_constraints_visibility(Operator, HighlightElement):
```

Comment:
Would be great to combine the operators and just have an enumproperty for the mode...

Reply:
Updated to use operator_enum.